install.sh

```diff
@@ -48,43 +48,7 @@ echo_intra() { echo -e "\e[1;34m${*}${RESET}" >&1; } # Use Blue for intrafunctio
 echo_out() { echo -e "\e[0;37m${*}${RESET}" >&1; } # Use Gray for program output.
 
 # Skip all checks if running on a docker container.
-if [[ -f "/.dockerenv" ]]; then
-  CREW_FORCE_INSTALL=1
-  echo_info "In container, so skipping hardware checks"
-fi
-
-#
-# Attempt install without sudo.
-#elif tty | grep -q /dev/pts1; then
-#  echo_success "You are in VT-2."
-#  if [ "${EUID}" == "0" ]; then
-#    echo_info "Please login as 'chronos' and restart the install."
-#    echo_info "Note that you can set the chronos password if you login as root and run 'chromeos-setdevpasswd'."
-#    exit 1
-#  fi
-#else
-#  # Error out if not in VT-2
-#  echo_error "You need to be in the VT-2 shell for the install to work."
-#  echo_info "Please use Ctrl-Alt-{F2/Right arrow/Refresh}) to switch to VT-2."
-#  echo_info "Then login as 'chronos' and restart the install."
-#  echo_info "Note that you can set the chronos password if you login as root and run 'chromeos-setdevpasswd'."
-#  exit 1
-#fi
-
-# Check if the script is being run as root.
-if [ "${EUID}" == "0" ]; then
-  echo_error "Chromebrew should not be installed or run as root."
-  exit 1;
-fi
-
-# Uncomment this section if you have a stoneyridge machine and your
```

Comment:
Please also leave this stoney ridge stuff in in some capacity so people still get the error message.